test/layer.yml

```diff
@@ -8,17 +8,16 @@ Parameters:
     Type: String
 Resources:
   Layer:
-    Type: 'AWS::CloudFormation::Stack'
+    Type: 'AWS::Lambda::LayerVersion'
     Properties:
-      Parameters:
-        ContentBucket: !Ref ContentBucket
-        ContentKey: !Ref ContentKey
-      TemplateURL: './node_modules/@cfn-modules/lambda-layer/module.yml'
+      Content:
+        S3Bucket: !Ref ContentBucket
+        S3Key: !Ref ContentKey
   Function:
     Type: 'AWS::CloudFormation::Stack'
     Properties:
       Parameters:
         Handler: 'defaults.handler'
         Runtime: 'nodejs8.10'
-        LayerModule: !GetAtt 'Layer.Outputs.StackName'
+        LayerArns: !Ref Layer
```

Comment:
Can you remove the lambda-layer dependency from test/package.json and run npm i again to update tets/package-lock.json as well?

Reply:
✅